### PR TITLE
chore: merge main into ai feature branch

### PR DESCRIPTION
**Pull request by atierian**
#### Description of changes
- new branch off of `feature/raven`
- merged `main` into it.
- accepted `main` changes.
- updated dependency versions in generation and conversation transformers.
- yarn && yarn package
- cut this PR
...
- removed fixjestenvironment from generation and conversation transformers
- updated generation and conversation snapshots for new jest formatting

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] Any CDK or CloudFormation parameter changes are called out explicitly

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by atierian** [comment]
enabled auto-merge as merge commit

**Review by dpilch — DISMISSED** [review]

File: packages/amplify-data-construct/.jsii [hunk 1]
```diff
@@ -10,11 +10,9 @@
     "@aws-amplify/backend-output-schemas": "^0.4.0",
     "@aws-amplify/backend-output-storage": "^0.2.2",
     "@aws-amplify/graphql-auth-transformer": "4.0.0",
-    "@aws-amplify/graphql-conversation-transformer": "0.1.0",
```
Comment (phani-srikar) [inline packages/amplify-data-construct/.jsii:13]: This is not expected right? You might need to refresh these files.

Comment (phani-srikar) [reply packages/amplify-data-construct/.jsii:13]: `yarn clean && yarn && yarn production-build` will give you any updated yarn.lock or jsii files.

Comment (atierian) [reply packages/amplify-data-construct/.jsii:13]: Thanks. I did `yarn && yarn build && yarn package` in [e59e763](https://github.com/aws-amplify/amplify-category-api/pull/2845/commits/e59e763d416cc2bbbd2d3e14dd79d98a9b85fb08). I think that did the trick, but I'm trying those commands now to see if there's a diff.

**Review by dpilch — APPROVED** [review]

**Review by phani-srikar — APPROVED** [review]

File: packages/amplify-data-construct/.jsii (hunk 1 above)
Comment (atierian) [reply packages/amplify-data-construct/.jsii:13]: No diff 😄

**Review by palpatim — APPROVED** [review]

**Pull request merged**